### PR TITLE
Feature/orbit databrew extension

### PR DESCRIPTION
**Pull request by stthoom**
### Description:

Orbit Jupyter-User JupyterLab Glue databrew extension. 
Included databrew demo notebook.

### Issue Reference URL

https://github.com/awslabs/aws-eks-data-maker/issues/<NUMBER>

----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS Orbit Workbench.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a ISSUE associated with this PR?
- [ ] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically 'main')?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0?

### CODE REVIEW

File: samples/manifests/plugins/dev-env-with-plugins.yaml [hunk 1]
```diff
@@ -8,8 +8,6 @@ UserPoolId: !SSM ${/orbit-foundation/dev-env/resources::UserPoolId}
 SharedEfsFsId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsFsId}
 SharedEfsSgId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsSgId}
 InstallSsmAgent: true
-CognitoExternalProvider: !ENV ${COGNITO_EXTERNAL_PROVIDER::}
```
Comment (stthoom) [inline samples/manifests/plugins/dev-env-with-plugins.yaml:11]: @rb201 - Please review this change. I was able to deploy by removing cognito attributes from manifest file.

File: samples/manifests/minimal/dev-env.yaml [hunk 2]
```diff
@@ -8,8 +8,6 @@ UserPoolId: !SSM ${/orbit-foundation/dev-env/resources::UserPoolId}
 SharedEfsFsId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsFsId}
 SharedEfsSgId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsSgId}
 InstallSsmAgent: true
-CognitoExternalProvider: !ENV ${COGNITO_EXTERNAL_PROVIDER::}
```
Comment (abaror) [inline samples/manifests/minimal/dev-env.yaml:11]: don't do this. Ramon will fix this properly

File: samples/manifests/plugins/dev-env-single-team.yaml [hunk 3]
```diff
@@ -8,8 +8,6 @@ UserPoolId: !SSM ${/orbit-foundation/dev-env/resources::UserPoolId}
 SharedEfsFsId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsFsId}
 SharedEfsSgId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsSgId}
 InstallSsmAgent: true
-CognitoExternalProvider: !ENV ${COGNITO_EXTERNAL_PROVIDER::}
```
Comment (abaror) [inline samples/manifests/plugins/dev-env-single-team.yaml:11]: same here .. don't change them.  Ramon will put a fix that if its empty its not doing it

File: images/jupyter-user/bundle.sh [hunk 4]
```diff
@@ -39,3 +39,14 @@ touch ${ARCHIVE_DIR}/pip.conf
...
 tar czvf aws-orbit_jupyter-user.tar.gz ./aws-orbit_jupyter-user
+
+
+# Cloning forked jupyterlab databrew extension
+git clone https://github.com/stthoom/aws-glue-databrew-jupyter-extension.git
+cd aws-glue-databrew-jupyter-extension
+git branch
+git checkout feature/orbit_databrew_extension
+cd ${DIR}
+tar czvf aws-glue-databrew-jupyter-extension.tar.gz ./aws-glue-databrew-jupyter-extension
```
Comment (chamcca) [inline images/jupyter-user/bundle.sh:50]: why do we bundle this as a .tar.gz into the image and not install it directly?

**Review by chamcca — APPROVED** [review]

File: samples/manifests/plugins/dev-env-single-team.yaml (hunk 3 above)
Comment (stthoom) [reply samples/manifests/plugins/dev-env-single-team.yaml]: Added cognito attributes to manifest files.

File: samples/manifests/minimal/dev-env.yaml (hunk 2 above)
Comment (stthoom) [reply samples/manifests/minimal/dev-env.yaml]: Added cognito attributes to manifest files.

File: samples/manifests/plugins/dev-env-with-plugins.yaml (hunk 1 above)
Comment (stthoom) [reply samples/manifests/plugins/dev-env-with-plugins.yaml]: Added cognito attributes to manifest files.

File: images/jupyter-user/bundle.sh (hunk 4 above)
Comment (stthoom) [reply images/jupyter-user/bundle.sh:50]: Thought of avoiding cloning and branch checkout steps(layers) in docker file. Will be replaced with "pip install " once we get JL 3 compatible databrew extension with pip packaging capability. As of now it is JL2 compatible and npm based. We forked and installing directly from source with JL version changes.

**Review by abaror — APPROVED** [review]

**Pull request merged**